### DISCUSSION TITLE
SignalK default username and password

### DISCUSSION THREAD

**Discussion by natdog83** [bareboat-necessities/lysmarine_gen#458] [Q&A] [answered]
Hello,  

I just installed the most stable version.  I'm trying to set up my connections in signalk but can't find the default username and password.  Does anyone know where I can find that.

**Comment by mgrouch** [comment, marked as answer]
There is no default. User name and password for SignalK is set by you on first login.  

The latest version on cloudsmith is the most stable. You can use it. 2024-10-07

**Reply by natdog83** [reply]
Thanks.  I'm not sure what happened but it's asking me for a username and password.  I'll try a couple before imaging the card and starting again.

**Reply by mgrouch** [reply]
First time you get to choose the user name and password. Next time you will have to use what you have chosen.

**Reply by natdog83** [reply]
Thanks.  I was able to figure it out with that help.
